oneview_redfish_toolkit/blueprints/resource_block_collection.py

```diff
@@ -14,23 +14,19 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import logging
-
-from flask import abort
 from flask import Blueprint
 from flask import g
-from flask import Response
-from flask_api import status
 
 from oneview_redfish_toolkit.api.resource_block_collection \
     import ResourceBlockCollection
-
+from oneview_redfish_toolkit.blueprints.util.response_builder import \
+    ResponseBuilder
 
 resource_block_collection = Blueprint("resource_block_collection", __name__)
 
 
 @resource_block_collection.route(
-    "/redfish/v1/CompositionService/ResourceBlocks/", methods=["GET"])
+    "/redfish/v1/CompositionService/ResourceBlocks", methods=["GET"])
```

Comment:
* If we remove the last `/`, a GET request on `/redfish/v1/CompositionService/ResourceBlocks/` will fail with status 404.
* If we keep the URI with the `/`, a request to `/redfish/v1/CompositionService/ResourceBlocks` (without the last `/`) will be redirected (301) to the correct URI